### DISCUSSION TITLE
Build failed in ios

### DISCUSSION THREAD

**Discussion by NgVThangBz** [axmolengine/axmol#1342] [Q&A] [answered]
im using axmol 2.0
use: "axmol build -p ios -a x64"
and when build in xcode 14.3.1 , Ipad th 7
Did I choose the wrong target or build something wrong?
![Screenshot 2023-09-20 at 11 07 16](https://github.com/axmolengine/axmol/assets/125545428/11cd02de-11a3-4038-ad4b-9a32ccdde06d)
![Screenshot 2023-09-20 at 11 07 50](https://github.com/axmolengine/axmol/assets/125545428/5e9d9c47-beb9-47d8-ad8b-011c4ae7b197)

**Comment by halx99** [comment, marked as answer]
device? not simulator?, `-a x64` is only for ios simulator, if you need build for device please use `-a arm64`

**Reply by NgVThangBz** [reply]
maybe I misunderstood, but in devSetup.md there is no "axmol build -p ios -a arm64"
and now it work fine, 
thank you
